src/ServiceControlInstaller.Engine.UnitTests/Validation/SelfContainedRavenDBTest.cs

```diff
@@ -15,16 +15,16 @@ public void CheckForSelfContainedRavenDB()
 
             var ravenServerPath = Path.GetFullPath(Path.Combine(TestContext.CurrentContext.TestDirectory, "..", "..", "..", "..", "..", "deploy", "RavenDBServer"));
             var ravenStudio = Path.Combine(ravenServerPath, "Raven.Studio.zip");
-            var runtimes = Path.Combine(ravenServerPath, "runtimes");
-            var systemCollections = Path.Combine(ravenServerPath, "System.Collections.dll");
-            var aspNetCoreHttp = Path.Combine(ravenServerPath, "Microsoft.AspNetCore.Http.dll");
+            var ravenServerDll = Path.Combine(ravenServerPath, "Raven.Server.dll");
+            var ravenServerExe = Path.Combine(ravenServerPath, "Raven.Server.exe");
+            var offlineOperationsUtility = Path.Combine(ravenServerPath, "rvn.exe");
 
             Assert.Multiple(() =>
             {
                 Assert.That(ravenStudio, Does.Exist); // No matter what
-                Assert.That(Directory.Exists(runtimes), Is.EqualTo(isLocal));  // Only in local development
-                Assert.That(File.Exists(systemCollections), Is.EqualTo(isCI)); // Only on CI
-                Assert.That(File.Exists(aspNetCoreHttp), Is.EqualTo(isCI));    // Only on CI
+                Assert.That(Directory.Exists(ravenServerDll), Is.EqualTo(isLocal));  // Only in local development
```

Comment:
_**Directory**_.Exists("some-path\Raven.Server.dll") ? Directory.Exists to a DLL file?

Reply:
Yeah, should have updated that as part of the changes I made to the test. Fixed.